sdk/include/opentelemetry/sdk/metrics/metric_reader.h

```diff
@@ -4,33 +4,73 @@
 #pragma once
 #ifndef ENABLE_METRICS_PREVIEW
 #  include <chrono>
+#  include "opentelemetry/sdk/common/global_log_handler.h"
+#  include "opentelemetry/sdk/metrics/instruments.h"
 #  include "opentelemetry/version.h"
 
 OPENTELEMETRY_BEGIN_NAMESPACE
 namespace sdk
 {
 namespace metrics
 {
+
 /**
  * MetricReader defines the interface to collect metrics from SDK
  */
 class MetricReader
 {
 public:
-  virtual ~MetricReader() = default;
+  MetricReader(
+      AggregationTemporarily aggregation_temporarily = AggregationTemporarily::kCummulative)
+      : aggregation_temporarily_(aggregation_temporarily), measurement_processor_callback_({})
+  {}
+
+  virtual ~MetricReader() { measurement_processor_callback_ = {}; }
```

Comment:
Why do we need this?

Reply:
Good catch. We don't need it. Removed now.